src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -535,6 +537,30 @@ export default {
 				'EXP-CASH-676-Apr2019',
 				'b',
 			);
+		};
+
+		// CASH-1801: recommendationChannel experiment setup
```

Comment:
Please break this out into it's own method like we did for `initializeFavoriteCountryRowExp` which will be your example to follow for this new row too :)

Reply:
Thanks for the tip! refactored the const into a function initializeRecommendedLoansRowExp()